pkg/install/images/images.go

```diff
@@ -202,6 +202,11 @@ func extractAddonsFromArchive(dir string, reader *tar.Reader) error {
 
 		path := filepath.Join(dir, header.Name)
 
+		// prevent path traversal (https://cwe.mitre.org/data/definitions/22.html)
+		if strings.Contains(path, "..") {
```

Comment:
How about stings like `\.\.` ? g

Reply:
Nevermind, I just checked and go complains about `unknown escape sequence` for an example above.

Reply:
This is also literally the suggested fix from the security advisory :grin: